lib/dataset/utils/parsers.js

```diff
@@ -48,7 +48,7 @@ function parseInterval(){
       .type('interval');
     each(res.result, function(record, i){
       var index = options[0] && options[0] === 'timeframe.end' ? record.timeframe.end : record.timeframe.start;
-      dataset.set(['Result', index], record.value);
+      dataset.set(['Result', String(index)], record.value);
```

Comment:
General question about this file: doesn't `datatset.set` already coerce the input coordinates into `string`s on line 54 and 60 of `dataset/index.js`? Is it necessary to keep casting the input into strings?